data.sql

```diff
@@ -22,3 +22,39 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto', '2022-05-14', 4, True, 22);
 
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34);
+
+INSERT INTO owners (full_name, age) VALUES ('Jennifer Orwell', 19);
+
+INSERT INTO owners (full_name, age) VALUES ('Bob', 45);
+
+INSERT INTO owners (full_name, age) VALUES ('Melody Pond', 77);
+
+INSERT INTO owners (full_name, age) VALUES ('Dean Winchester', 14);
+
+INSERT INTO owners (full_name, age) VALUES ('Jodie Whittaker', 38);
+
+INSERT INTO species (name) VALUES ('Pokemon');
+
+INSERT INTO species (name) VALUES ('Digimon');
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith')
+WHERE name = 'Agumon';
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell')
+WHERE name IN ('Gabumon', 'Pikachu');
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob')
+WHERE name IN ('Devimon', 'Plantmon');
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond')
+WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
+WHERE name IN ('Angemon', 'Boarmon');
+
```

Comment:
- [ ] Good job working on these, but you are still missing some commands. Please kindly ensure you include commands for the query below.
`if an animal's name ends with mon it should have Digimon as a species_id, all other animals should have Pokemon instead`